src/components/ButtonPanel.js

```diff
@@ -1,42 +1,52 @@
 import React from 'react';
+import PropTypes from 'prop-types';
 import Button from './Button';
 
-export default function ButtonPanel() {
+export default function ButtonPanel({ clickHandler }) {
   return (
     <div className="groups">
       <div className="G1">
-        <Button name="AC" />
-        <Button name="+/-" />
-        <Button name="%" />
-        <Button name="/" />
+        <Button name="AC" handleClick={clickHandler} />
+        <Button name="+/-" handleClick={clickHandler} />
+        <Button name="%" handleClick={clickHandler} />
+        <Button name="/" handleClick={clickHandler} />
       </div>
 
       <div className="G2">
-        <Button name="7" />
-        <Button name="8" />
-        <Button name="9" />
-        <Button name="*" />
+        <Button name="7" handleClick={clickHandler} />
+        <Button name="8" handleClick={clickHandler} />
+        <Button name="9" handleClick={clickHandler} />
+        <Button name="*" handleClick={clickHandler} />
       </div>
 
       <div className="G3">
-        <Button name="4" />
-        <Button name="5" />
-        <Button name="6" />
-        <Button name="-" />
+        <Button name="4" handleClick={clickHandler} />
+        <Button name="5" handleClick={clickHandler} />
+        <Button name="6" handleClick={clickHandler} />
+        <Button name="-" handleClick={clickHandler} />
       </div>
 
       <div className="G4">
-        <Button name="3" />
-        <Button name="2" />
-        <Button name="1" />
-        <Button name="+" />
+        <Button name="1" handleClick={clickHandler} />
+        <Button name="2" handleClick={clickHandler} />
+        <Button name="3" handleClick={clickHandler} />
+        <Button name="+" handleClick={clickHandler} />
       </div>
 
       <div className="G5">
-        <Button name="0" />
-        <Button name="." />
-        <Button name="=" />
+        <Button name="0" handleClick={clickHandler} />
+        <Button name="." handleClick={clickHandler} />
+        <Button name="=" handleClick={clickHandler} />
       </div>
     </div>
   );
 }
+
+ButtonPanel.propTypes = {
+  clickHandler: PropTypes.func.isRequired,
+};
+
+// eslint-disable-next-line no-unused-vars
```

Comment:
Kindly remove the inline Eslint comment before merging 👍 
👀  on you!

It would be nice to arrange your codebase so that you won't have to use inline Eslint comment, which Microverse as a whole doesn't encourage you to use.

Head-Ups: If use inline Eslint in the next milestone, 99% chance it won't be approved.